adafruit_displayio_layout/widgets/icon_widget.py

```diff
@@ -52,19 +64,67 @@ class IconWidget(Widget, Control):
     :param int max_size: (Optional) this will get passed through to the
      displayio.Group constructor. If omitted we default to
      grid_size width * grid_size height to make room for all (1, 1) sized cells.
+    :param int wheel_initial_value: When using palette animation, this is the initial value
+     of the colorwheel parameter used with ``_pixelbuf.colorwheel``
+    :param int wheel_increment: To add palette animation, set this to the value of
+     how much you want the ``_pixelbuf.colorwheel`` function to increment each time that
+     ``unselected`` is called (default: 0 for no palette animation)
+    :param int wheel_grading: This is the step sized used when calling colorwheel for
+     each color index in the palette (default: 5), basically it's how far apart each
+     color in the palette will be set. Use a low value if you want each color to be
+     close to each other or a high value to spread out into a wider range of colors.
+    :param int palette_skip_indices: integer or list of integers with the palette
+     indices that should not be changed when using the palette animations (default: None)
 
     """
 
-    def __init__(self, label_text, icon, on_disk=False, **kwargs):
-        super().__init__(**kwargs)
+    # pylint: disable=bad-super-call, too-many-instance-attributes, too-many-locals
+    # pylint: disable=too-many-arguments, unused-argument
+
+    _max_scale = 1.5
+    _max_pixels = 80
+    _max_color_depth = 512
+    _zoom_bitmap = Bitmap(
+        round(_max_scale * _max_pixels),
+        round(_max_scale * _max_pixels),
+        _max_color_depth,
+    )
+    _zoom_palette = Palette(_max_color_depth)
+
+    def __init__(
+        self,
+        display,
+        label_text,
+        icon,
+        on_disk=False,
+        max_scale=1.0,
+        max_angle=8,
+        animation_time=0.0,
+        wheel_initial_value=1,  # initial wheel color value
+        wheel_increment=0,  # how much the wheel
+        wheel_grading=5,  # sets the colorwheel distance between palette colors
+        palette_skip_indices=None,  # single value or list of palette color indices to
+        # remain constant during animations
```

Comment:
this comment line feels like it should be indented over to the right underneath the comment from the line above to me.

But black and pylint both ran without complaining so perhaps my feeling is off on this one.